Mac OS/X 32 bit build broken at commit 1a2cc27e013b561c9d3c8b81384d14443822057f

Mac 32 bit build now fails with the following error:

```
db/column_family.cc:152:38: error: non-type template argument evaluates to 68719476736, which cannot be narrowed to type 'unsigned long' [-Wc++11-narrowing]
      std::integral_constant<size_t, 64ull << 30>>::type::value;
                                     ^
1 error generated.
```

--- a/db/column_family.cc
+++ b/db/column_family.cc
@@ -149,7 +149,7 @@ ColumnFamilyOptions SanitizeOptions(const DBOptions& db_options,
   result.comparator = icmp;
   size_t clamp_max = std::conditional<
       sizeof(size_t) == 4, std::integral_constant<size_t, 0xffffffff>,
-      std::integral_constant<size_t, 64ull << 30>>::type::value;
+      std::integral_constant<uint64_t, 64ull << 30>>::type::value;
   ClipToRange(&result.write_buffer_size, ((size_t)64) << 10, clamp_max);
   // if user sets arena_block_size, we trust user to use this value. Otherwise,
   // calculate a proper value from writer_buffer_size;

